### PR TITLE
Doc 1778 v2

### PR DESCRIPTION
**Pull request by regit**
Update of #4250 addressing comments

- [x] I have read the contributing guide lines at https://redmine.openinfosecfoundation.org/projects/suricata/wiki/Contributing
- [x] I have signed the Open Information Security Foundation contribution agreement at https://suricata-ids.org/about/contribution-agreement/
- [x] I have updated the user guide (in doc/userguide/) to reflect the changes made (if applicable)

Link to [redmine](https://redmine.openinfosecfoundation.org/projects/suricata/issues) ticket:
- https://redmine.openinfosecfoundation.org/issues/1778
- https://redmine.openinfosecfoundation.org/issues/2876

Describe changes:
- Address comments

[PRScript](https://redmine.openinfosecfoundation.org/projects/suricata/wiki/PRscript) output:
- PR regit: https://buildbot.openinfosecfoundation.org/builders/regit/builds/500
- PR regit-pcap: https://buildbot.openinfosecfoundation.org/builders/regit-pcap/builds/276

### CODE REVIEW

File: doc/userguide/setting-up-ipsinline-for-linux.rst [hunk 1]
```diff
@@ -115,12 +118,134 @@ There is also a way to use iptables with multiple networks (and interface cards)
...
+To only do it for packets exchanged between eth0 and eth1 ::
+
+ nft> add rule filter IPS iif eth0 oif eth1 queue
+ nft> add rule filter IPS iif eth1 oif eth0 queue
+
+NFQUEUE advanced options
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+NFQUEUE mechanism supports some interesting options. The nftables configuration will be shown there
+but the feature are also available in iptables.
```
Comment (norg) [inline doc/userguide/setting-up-ipsinline-for-linux.rst]: feature*s*

**Comment by norg** [comment]
In general LGTM

File: doc/userguide/setting-up-ipsinline-for-linux.rst (hunk 1 above)
Comment (victorjulien) [reply doc/userguide/setting-up-ipsinline-for-linux.rst]: Fixing up during merge.

**Comment by victorjulien** [comment]
Merged in #4271, thanks Eric. I did some reformatting on top of this.

**Pull request closed without merging**